### PR TITLE
Change 3 instances of "is" to "==" per syntax warning

### PR DESCRIPTION
**Pull request by shielddigitaldesign**
Re-try. See comment on previous pull request, feel free to reject if needed.

### CODE REVIEW

**Comment by PetePupalaikis** [comment]
I did manage to get this on a local branch from InNextRelease.

I ran the test suite and one test fails:

In **TestChirpZTransform.py**:
`.E...
======================================================================
ERROR: testSumOfSines4 (__main__.TestChirpZTransform)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "TestChirpZTransform.py", line 207, in testSumOfSines4
    wf=si.td.wf.SineWaveform(td,Frequency=fd[n],Phase=90.).__radd__(wf)
  File "/home/petep/Work/SignalIntegrity/SignalIntegrity/Lib/TimeDomain/Waveform/Waveform.py", line 161, in __radd__
    if other == 0:
  File "/home/petep/Work/SignalIntegrity/SignalIntegrity/Lib/TimeDomain/Waveform/Waveform.py", line 268, in __eq__
    if len(self) != len(other):
TypeError: object of type 'int' has no len()

----------------------------------------------------------------------
Ran 25 tests in 1.005s

FAILED (errors=1)
`

**Comment by PetePupalaikis** [comment]
I think I like the is 0 check.  I think this is standard with __radd__ to avoid a common problem, which is that of summing objects together.  (If I remember correctly a sum of waveforms starts with 0 added to a waveform, hence the __radd__ code the way it was written.

File: SignalIntegrity/Lib/TimeDomain/Waveform/Waveform.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def __radd__(self, other):
         @return self+other
         @see Waveform.__add__()
         """
-        if other is 0:
+        if other == 0:
```
Comment (PetePupalaikis) [inline SignalIntegrity/Lib/TimeDomain/Waveform/Waveform.py]: I think we don't want this change and to leave the original as is.

**Comment by shielddigitaldesign** [comment]
Hi Pete,

I agree that the change isn't needed now that you've explained the reasoning. I think it's best to reject the whole pull request, but at least now I've got the flow down. Any recommendations on what parts to work on?

-Stephen

**Pull request closed without merging**